sdk/aqueduct/constants/enums.py

```diff
@@ -100,6 +100,7 @@ class RelationalDBServices(str, Enum, metaclass=MetaEnum):
 class ExecutionStatus(str, Enum, metaclass=MetaEnum):
     UNKNOWN = "unknown"
     SUCCEEDED = "succeeded"
+    RUNNING = "running"
```

Comment:
We're missing this entry, which is why pydantic was complaining on the SDK side for the preview response.